src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE projectDeliveryEmployees (
```

Comment:
whats the naming convention for the file?

Reply:
changed